packages/runtime/src/widgets/Search.tsx

```diff
@@ -18,31 +24,46 @@ export type SearchProps = {
   placeholder?: string;
   data?: Expression<object>;
   searchKey?: string;
+  onValueChange?: EnsembleAction;
+  onSelect?: EnsembleAction;
 } & EnsembleWidgetProps<SearchStyles>;
 
 export const Search: React.FC<SearchProps> = ({
+  id,
   placeholder,
   data,
   searchKey,
+  onValueChange,
+  onSelect,
   styles,
 }) => {
   const [options, setOptions] = useState<SelectProps<object>["options"]>([]);
+  const [value, setValue] = useState("");
+  // eslint-disable-next-line no-empty-pattern
+  const {} = useRegisterBindings(
```

Comment:
You can just call useRegisterBindings without assigning the result to anything to avoid the linter error